test/README.md

```diff
@@ -48,6 +48,25 @@ npm run lint:javascript
   
 ## Advanced Workflow
 
+#### View tests performing in realtime
+You can view the testing process as it happens in realtime with [Selenium Server](https://selenium.dev/downloads/) and using [VNC Viewer](https://www.realvnc.com/en/connect/download/viewer/).
+
+Such as an example workflow:
+
+1) Start the Selenium Server:
+    ```
+    java -jar selenium-server-standalone-2.53.1.jar
```

Comment:
Any idea whether this version is current and consistent with what Docker is using?

Reply:
There's definitely at least 3.x released, but I don't know what Docker is using.

Reply:
Any info about `java` requirement?
Oracle JDK or OpenJDK?
min Java version?

Reply:
Docker is running with selenium 3.8.1-aluminum

Reply:
1) Start the Selenium Server:     This line is unnecessary, The docker test has included the selenium server.